app/preload/index.ts

```diff
@@ -5,7 +5,7 @@ const { contextBridge, ipcRenderer } = require('electron');
 const electronContext: ElectronRendererContext = {
   onUpdate: callback => ipcRenderer.on('update', (_, event, data) => callback(event, data)),
 
-  initlizeUpdater: () => ipcRenderer.send('initlizeUpdater'),
+  initializeUpdater: () => ipcRenderer.send('initlizeUpdater'),
```

Comment:
```suggestion
  initializeUpdater: () => ipcRenderer.send('initializeUpdater'),
```
위 코드처럼 채널명도 바꾼뒤
`app/modules/update/index.ts` 에 리스너 채널명도 바꿔주세요